Kiwix-tools does not compile anymore against cutting edge libkiwix

```
[2/8] Compiling C++ object 'src/server/e19e62e@@kiwix-serve@exe/kiwix-serve.cpp.o'.
FAILED: src/server/e19e62e@@kiwix-serve@exe/kiwix-serve.cpp.o 
ccache c++ -Isrc/server/e19e62e@@kiwix-serve@exe -Isrc/server -I../src/server -I/usr/local/include/ -I/usr/local/include -I/usr/include/x86_64-linux-gnu -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Werror -std=c++11 -g '-DKIWIX_TOOLS_VERSION="3.1.2"' -pthread -MD -MQ 'src/server/e19e62e@@kiwix-serve@exe/kiwix-serve.cpp.o' -MF 'src/server/e19e62e@@kiwix-serve@exe/kiwix-serve.cpp.o.d' -o 'src/server/e19e62e@@kiwix-serve@exe/kiwix-serve.cpp.o' -c ../src/server/kiwix-serve.cpp
../src/server/kiwix-serve.cpp: In function ‘int main(int, char**)’:
../src/server/kiwix-serve.cpp:190:42: error: ‘split’ is not a member of ‘kiwix’; did you mean ‘kainjow::mustache::split’?
  190 |     vector<string> libraryPaths = kiwix::split(libraryPath, ";");
      |                                          ^~~~~
In file included from /usr/local/include/kiwix/tools/otherTools.h:27,
                 from ../src/server/kiwix-serve.cpp:24:
/usr/local/include/mustache.hpp:88:26: note: ‘kainjow::mustache::split’ declared here
   88 | std::vector<string_type> split(const string_type& s, typename string_type::value_type delim) {
      |                          ^~~~~
../src/server/kiwix-serve.cpp:199:17: error: ‘isRelativePath’ was not declared in this scope
  199 |               = isRelativePath(*itr)
      |                 ^~~~~~~~~~~~~~
../src/server/kiwix-serve.cpp:200:43: error: ‘getCurrentDirectory’ was not declared in this scope
  200 |                     ? computeAbsolutePath(getCurrentDirectory(), *itr)
      |                                           ^~~~~~~~~~~~~~~~~~~
../src/server/kiwix-serve.cpp:200:23: error: ‘computeAbsolutePath’ was not declared in this scope
  200 |                     ? computeAbsolutePath(getCurrentDirectory(), *itr)
      |                       ^~~~~~~~~~~~~~~~~~~
[4/8] Compiling C++ object 'src/manager/2de60dd@@kiwix-manage@exe/kiwix-manage.cpp.o'.
FAILED: src/manager/2de60dd@@kiwix-manage@exe/kiwix-manage.cpp.o 
ccache c++ -Isrc/manager/2de60dd@@kiwix-manage@exe -Isrc/manager -I../src/manager -I/usr/local/include/ -I/usr/local/include -I/usr/include/x86_64-linux-gnu -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Werror -std=c++11 -g '-DKIWIX_TOOLS_VERSION="3.1.2"' -pthread -MD -MQ 'src/manager/2de60dd@@kiwix-manage@exe/kiwix-manage.cpp.o' -MF 'src/manager/2de60dd@@kiwix-manage@exe/kiwix-manage.cpp.o.d' -o 'src/manager/2de60dd@@kiwix-manage@exe/kiwix-manage.cpp.o' -c ../src/manager/kiwix-manage.cpp
../src/manager/kiwix-manage.cpp: In function ‘int main(int, char**)’:
../src/manager/kiwix-manage.cpp:258:17: error: ‘isRelativePath’ was not declared in this scope
  258 |   libraryPath = isRelativePath(libraryPath)
      |                 ^~~~~~~~~~~~~~
../src/manager/kiwix-manage.cpp:259:43: error: ‘getCurrentDirectory’ was not declared in this scope
  259 |                     ? computeAbsolutePath(getCurrentDirectory(), libraryPath)
      |                                           ^~~~~~~~~~~~~~~~~~~
../src/manager/kiwix-manage.cpp:259:23: error: ‘computeAbsolutePath’ was not declared in this scope
  259 |                     ? computeAbsolutePath(getCurrentDirectory(), libraryPath)
      |                       ^~~~~~~~~~~~~~~~~~~
[5/8] Linking target src/searcher/kiwix-search.
ninja: build stopped: subcommand failed.
```

--- a/src/manager/kiwix-manage.cpp
+++ b/src/manager/kiwix-manage.cpp
@@ -18,8 +18,8 @@
  */
 
 #include <getopt.h>
-#include <kiwix/tools/stringTools.h>
 #include <kiwix/manager.h>
+#include <kiwix/tools.h>
 #include <cstdlib>
 #include <iostream>
 
@@ -255,8 +255,8 @@ int main(int argc, char** argv)
   }
 
   /* Try to read the file */
-  libraryPath = isRelativePath(libraryPath)
-                    ? computeAbsolutePath(getCurrentDirectory(), libraryPath)
+  libraryPath = kiwix::isRelativePath(libraryPath)
+                    ? kiwix::computeAbsolutePath(kiwix::getCurrentDirectory(), libraryPath)
                     : libraryPath;
   kiwix::Manager manager(&library);
   manager.readFile(libraryPath, false);

--- a/src/server/kiwix-serve.cpp
+++ b/src/server/kiwix-serve.cpp
@@ -21,7 +21,7 @@
 #include <kiwix/manager.h>
 #include <kiwix/server.h>
 #include <kiwix/name_mapper.h>
-#include <kiwix/tools/otherTools.h>
+#include <kiwix/tools.h>
 
 #ifdef _WIN32
 # include <windows.h>
@@ -196,8 +196,8 @@ int main(int argc, char** argv)
 
         try {
           string libraryPath
-              = isRelativePath(*itr)
-                    ? computeAbsolutePath(getCurrentDirectory(), *itr)
+              = kiwix::isRelativePath(*itr)
+                    ? kiwix::computeAbsolutePath(kiwix::getCurrentDirectory(), *itr)
                     : *itr;
           retVal = manager.readFile(libraryPath, true, trustlibrary);
         } catch (...) {

